### PR TITLE
Add maxtext based llama2-70b, mixstral-8x7b, gpt3-175b, llama3-405b

### PR DESCRIPTION
**Pull request by suexu1025**
* Add maxtext based llam2-70b, mixstral-8x7b, gpt3-175b benchmark on trillium
* Common readme for xpk/maxtext

### CODE REVIEW

File: training/trillium/GPT3-175B-MaxText/README.md
Comment (Obliviour) [inline training/trillium/GPT3-175B-MaxText/README.md:1]: @raymondzouu can you work with @suexu1025 on adding the multinic settings + daemon set needed for good multislice performance

Comment (raymondzouu) [reply training/trillium/GPT3-175B-MaxText/README.md:1]: @suexu1025 Can you use the network and multinic setup steps from http://shortn/_veTkDLz9yq and then the performance daemon set command from http://shortn/_IkvXopxx90

File: training/trillium/MAXTEXT_REAME.md
Comment (Obliviour) [inline training/trillium/MAXTEXT_REAME.md:1]: typo in file name

File: training/trillium/MAXTEXT_README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Prep for Maxtext GPT3-175B workloads on GKE
+1. Clone [Maxtext](https://github.com/google/maxtext) repo and move to its directory
+```
+git clone https://github.com/google/maxtext.git
+cd maxtext
```
Comment (parambole) [inline training/trillium/MAXTEXT_README.md:5]: Hey @suexu1025 can we mention here that we need to clone maxtext at a particular commit i.e. [maxtext](https://github.com/AI-Hypercomputer/maxtext.git)@2e1ebad7c660e45d2f020ef025d74cc90e2f0eb3. 

Given that we want to create the maxtext image from that hash.

File: training/trillium/GPT3-175B-MaxText/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+## Prep for Maxtext 
+Please follow this [link](https://github.com/AI-Hypercomputer/tpu-recipes/training/trillium/MAXTEXT_README.md) to install maxtext and build docker image
+
+## Run Maxtext GPT3-175B workloads on GKE
+
+### Test Env
+jaxlib=0.4. 35
+
+libtpu-nighly=20241028
```
Comment (parambole) [inline training/trillium/GPT3-175B-MaxText/README.md:14]: @suexu1025 Please correct typo libtpu-nighly > libtpu-nightly in this and other files.

Comment (parambole) [inline training/trillium/GPT3-175B-MaxText/README.md:14]: @suexu1025 The libtpu version is the one that is in the JAX stable stack (which is derived from the JAX version). Is there a specific reason we are mentioning this here and in other files that this is the test environment? I thought the mention of overriding the libtpu version to nightly was to get access to the latest and experimental features.

Comment (suexu1025) [reply training/trillium/GPT3-175B-MaxText/README.md]: stable stack used libtpu`0.1.dev20241010+nightly.cleanup` and we noticed non-trival performance improvement using this newer libtpu and matching with launch pad.

File: training/trillium/GPT3-175B-MaxText/README.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+## XPK setup
+Please follow this [link](https://github.com/AI-Hypercomputer/tpu-recipes/training/trillium/XPK_README.md) to create your GKE cluster with XPK
+
+## Prep for Maxtext 
+Please follow this [link](https://github.com/AI-Hypercomputer/tpu-recipes/training/trillium/MAXTEXT_README.md) to install maxtext and build docker image
+
+## Run Maxtext GPT3-175B workloads on GKE
+
+### Test Env
+jaxlib=0.4. 35
```
Comment (yacoub-google) [inline training/trillium/GPT3-175B-MaxText/README.md:12]: Nit: is there supposed to be a space before 35?

File: training/trillium/MAXTEXT_README.md (hunk 1 above)
Comment (suexu1025) [reply training/trillium/MAXTEXT_README.md:5]: Added

File: training/trillium/GPT3-175B-MaxText/README.md (hunk 3 above)
Comment (suexu1025) [reply training/trillium/GPT3-175B-MaxText/README.md]: nope, I removed it.

File: training/trillium/MAXTEXT_README.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
+# Prep for Maxtext GPT3-175B workloads on GKE
```
Comment (yacoub-google) [inline training/trillium/MAXTEXT_README.md:1]: is this specific to GPT3-175? Seems like it applies to the other recipes, so should we delete "GPT3-175B" from this line?

**Review by ultrons — APPROVED** [review]

**Pull request merged**